src/main/java/hudson/tasks/test/TestObject.java

```diff
@@ -199,7 +200,7 @@ public String getRelativePathFrom(TestObject it) {
                 buf.insert(0, hudsonRootUrl);
             }
 
-            LOGGER.info("Here's our relative path: " + buf.toString()); 
+            LOGGER.log(Level.FINE, "Here is our relative path: {0}", buf);
```

Comment:
```suggestion
            LOGGER.fine(() -> "Here is our relative path: " + buf);
```

Reply:
You can use the lambda form too. In my tests it is slightly more expensive for one argument, slightly less for two or more. The bytecode is a bit longer. :man_shrugging: